JSONResponse.cpp

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "JSONResponse.h"
+
+namespace facebook {
+namespace graphql {
+namespace rapidjson {
+
+Value convertResponse(Document::AllocatorType& allocator, response::Value&& response)
```

Comment:
Just a thought.
With the SAX Writer API, I'm not sure it is necessary to convert to a RapidJSON Document/Value:
- http://rapidjson.org/md_doc_sax.html#Writer

However,

Reply:
Good point. As is, the JSON conversion is decoupled from the GraphQL responses internally. I was thinking of this as more of a reference sample to encourage swapping in another JSON library.

But most consumers (including my tests) are going to want to convert to JSON eventually, and they might as well use this implementation. Installing it to the include directory and moving the JSON library dependency out of the header and into another target library (using the more efficient SAX writer) would make that easier to consumer.

This might also be a good time to spin up a contrib directory and accept alternate JSON library bindings.

Reply:
Added #14 to track that.